[2024-11-12T23:24:43.794+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-12T23:24:44.002+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: brew-dag.e_valido scheduled__2024-11-11T07:00:00+00:00 [queued]>
[2024-11-12T23:24:44.073+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: brew-dag.e_valido scheduled__2024-11-11T07:00:00+00:00 [queued]>
[2024-11-12T23:24:44.074+0000] {taskinstance.py:2866} INFO - Starting attempt 4 of 5
[2024-11-12T23:24:44.098+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): e_valido> on 2024-11-11 07:00:00+00:00
[2024-11-12T23:24:44.109+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=722) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-12T23:24:44.111+0000] {standard_task_runner.py:72} INFO - Started process 724 to run task
[2024-11-12T23:24:44.111+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'brew-dag', 'e_valido', 'scheduled__2024-11-11T07:00:00+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/brew-dag.py', '--cfg-path', '/tmp/tmp029it1xe']
[2024-11-12T23:24:44.113+0000] {standard_task_runner.py:105} INFO - Job 15: Subtask e_valido
[2024-11-12T23:24:44.172+0000] {task_command.py:467} INFO - Running <TaskInstance: brew-dag.e_valido scheduled__2024-11-11T07:00:00+00:00 [running]> on host 60459ae802f2
[2024-11-12T23:24:44.265+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='brew-dag' AIRFLOW_CTX_TASK_ID='e_valido' AIRFLOW_CTX_EXECUTION_DATE='2024-11-11T07:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-11T07:00:00+00:00'
[2024-11-12T23:24:44.267+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-12T23:24:44.269+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-12T23:24:44.270+0000] {logging_mixin.py:190} INFO - Current task name:e_valido state:running start_date:2024-11-12 23:24:44.055476+00:00
[2024-11-12T23:24:44.271+0000] {logging_mixin.py:190} INFO - Dag name:brew-dag and current dag run status:running
[2024-11-12T23:24:44.272+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-12T23:24:44.273+0000] {baseoperator.py:416} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-11-12T23:24:44.286+0000] {python.py:240} INFO - Done. Returned value was: valido
[2024-11-12T23:24:44.287+0000] {branch.py:38} INFO - Branch into valido
[2024-11-12T23:24:44.288+0000] {skipmixin.py:230} INFO - Following branch ('valido',)
[2024-11-12T23:24:44.305+0000] {skipmixin.py:278} INFO - Skipping tasks [('nvalido', -1)]
[2024-11-12T23:24:44.344+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-12T23:24:44.345+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=brew-dag, task_id=e_valido, run_id=scheduled__2024-11-11T07:00:00+00:00, execution_date=20241111T070000, start_date=20241112T232444, end_date=20241112T232444
[2024-11-12T23:24:44.359+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-12T23:24:44.360+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-12T23:24:44.360+0000] {logging_mixin.py:190} INFO - Dag name:brew-dag queued_at:2024-11-12 23:24:14.838184+00:00
[2024-11-12T23:24:44.361+0000] {logging_mixin.py:190} INFO - Task hostname:60459ae802f2 operator:BranchPythonOperator
[2024-11-12T23:24:44.407+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-12T23:24:44.441+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-12T23:24:44.448+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
